### DISCUSSION TITLE
Since when can we use the extension without docker desktop

### DISCUSSION THREAD

**Discussion by extreme4all** [microsoft/vscode-docker#4373] [Q&A]
i used to have docker desktop open (windows machine) for the extension to see my docker containers, this is no longer the case YAY!!!
i double checked this on my linux (ubuntu) laptop where i completely uninstalled docker & docker desktop and reinstalled it with the get docker script (https://github.com/docker/docker-install) and it works, it finally works without docker desktop.

my question since when is this possible?

**Comment by bwateratmsft** [comment]
Docker Desktop has never been a strict requirement for the VSCode Docker extension. In the past we used a mix of the CLI and the HTTP endpoint to communicate with Docker, but since ~1.23 we have exclusively used the CLI.
